### ISSUE TITLE
Demo link

### ISSUE THREAD

**Issue by jasonmelgoza** [paulstraw/sundial#1]
Would be nice to see a demo and add a link to it on the readme.

**Comment by paulstraw** [comment]
Absolutely! A nice demo is 100% on my list, but in the meantime, here's this lame test bed I used while developing it: http://preview.octopuscreative.com/sundial/

**Comment by jasonmelgoza** [comment]
@paulstraw Awesome! Looks great.
